packages/bam-ppx/bam-ppx.0.2/opam

```diff
@@ -0,0 +1,42 @@
+opam-version: "2.0"
+synopsis: "A PPX deriving generators for OCaml types"
+description:
+  "Provides a way to automatically get generators for a given type"
+maintainer: ["François Thiré"]
+authors: ["François Thiré"]
+license: "MIT"
+tags: ["test" "pbt" "shrinking" "internal" "bam" "ppx"]
+homepage: "https://github.com/francoisthire/bam"
+doc: "https://francoisthire.github.io/bam/"
+bug-reports: "https://github.com/francoisthire/bam/issues"
+depends: [
+  "ocaml" {>= "4.14" & < "5.2"}
+  "dune" {>= "3.7" & >= "3.7"}
```

Comment:
```suggestion
  "dune" {>= "3.7"}
```
This is fixed on the repo itself by removing `dune` from the dependency list in the dune-project file (it is added automatically)